src/main/groovy/com/benjaminsproule/swagger/gradleplugin/VersionUtils.groovy

```diff
@@ -0,0 +1,36 @@
+package com.benjaminsproule.swagger.gradleplugin
+
+import org.gradle.api.GradleException
+
+class VersionUtils {
+
+    def static ensureCompatibleSwaggerSpec() {
+        if (useSwaggerSpec11()) {
```

Comment:
We can combine this and the below and change the message to simply say they aren't supported